numpy/core/fromnumeric.py

```diff
@@ -2234,6 +2242,13 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue):
         sub-class' method does not implement `keepdims` any
         exceptions will be raised.
 
+    initializer : scalar, optional
+        The maximum value of an output element. Must be present to allow
```

Comment:
This actually becomes the minimum value the maximum will have, right? Maybe rephrase to make that clearer (and the same for `amin` below).

...

Must admit I cannot easily think of a better way of phrasing this. But at least change maximum -> minimum here.

Reply:
Actually, this is correct. It's the maximum value that the minimum can hold.